artifactory/commands/generic/download.go

```diff
@@ -212,6 +214,8 @@ func getDownloadParams(f *spec.File, configuration *utils.DownloadConfiguration)
 		return
 	}
 
+	downParams.Transitive = strings.ToLower(os.Getenv("JFROG_CLI_TRANSITIVE_DOWNLOAD_EXPERIMENTAL")) == "true"
```

Comment:
Let's use a constant for this env var, and put it in ```coreconsts.go``` together with the other env var consts. It's easier to have all the env vars available in one place.